tests/test_package.py

```diff
@@ -1107,3 +1107,273 @@ def test_package_pprint_1029():
     expected = """{'resources': [{'data': [['id', 'name'], ['1', 'english'], ['2', '中国人']],
                 'name': 'name'}]}"""
     assert repr(package) == expected
+
+
+def test_package_to_markdown_837():
+    descriptor = {
+        "name": "package",
+        "resources": [
+            {
+                "name": "main",
+                "schema": {
+                    "fields": [
+                        {
+                            "name": "id",
+                            "description": "Any positive integer",
+                            "type": "integer",
+                            "constraints": {"minimum": 1},
+                        },
+                        {
+                            "name": "integer_minmax",
+                            "description": "An integer between 1 and 10",
+                            "type": "integer",
+                            "constraints": {"minimum": 1, "maximum": 10},
+                        },
+                        {
+                            "name": "boolean",
+                            "description": "Any boolean",
+                            "type": "boolean",
+                        },
+                    ],
+                    "primaryKey": ["id"],
+                },
+            },
+            {
+                "name": "secondary",
+                "schema": {
+                    "fields": [
+                        {
+                            "name": "main_id",
+                            "description": "Any value in main.id",
+                            "type": "integer",
+                        },
+                        {
+                            "name": "string",
+                            "description": "Any string of up to 3 characters",
+                            "type": "string",
+                            "constraints": {"maxLength": 3},
+                        },
+                    ],
+                    "foreignKeys": [
+                        {
+                            "fields": ["main_id"],
+                            "reference": {"resource": "main", "fields": ["id"]},
+                        }
+                    ],
+                },
+            },
+        ],
+    }
+    package = Package(descriptor)
+    expected = """# `package`
```

Comment:
I would put this text and similar to `tests/fixtures/output-markdown` or something like this. It will be easy to read and maintain